### PR TITLE
Add carthage directory to git

### PR DESCRIPTION
**Pull request by scruff311**
We had to remove the TV targets from the SnapKit and MarqueeLabel dependencies to get the app building with Xcode 14 so we are now tracking the Carthage directory in git so this change can be commited.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,7 +22,7 @@ DerivedData
 # Bundler
 .bundle
 
-Carthage
+# Carthage
```
Comment (zhanfengh) [inline .gitignore:27]: Should we remove this line instead of commenting it out?

**Review by zhanfengh — COMMENTED** [review]
Just one question, otherwise, looking good!

File: .gitignore (hunk 1 above)
Comment (scruff311) [reply .gitignore:27]: I added a comment explaining why we made the change instead. Does that work for you?

Comment (zhanfengh) [reply .gitignore:27]: Nice! that's even better

**Review by zhanfengh — APPROVED** [review]
👏 🚢

**Review by jpcamara — APPROVED** [review]

**Pull request merged**